packages/flutter_tools/test/integration.shard/test_data/gen_l10n_project.dart

```diff
@@ -94,7 +94,7 @@ class Home extends StatelessWidget {
     final List<String> results = [];
     return Row(
       children: <Widget>[
-        ResultBuilder(
+        LocaleBuilder(
```

Comment:
Are the changes in this file related to deferred loading?

Reply:
Yes. It was quite hard to pin this down. If you look at the description of the PR you can see the full details.

The TLDR is that this change is needed as the `Localizations` widget rebuilds itself after localizations have loaded. This change ensure that all of the printing widgets in the test will wait for all localizations to load first.